packages/cli/src/workflows/workflow.request.ts

```diff
@@ -42,7 +42,12 @@ export declare namespace WorkflowRequest {
 
 	type NewName = AuthenticatedRequest<{}, {}, {}, { name?: string }>;
 
-	type ManualRun = AuthenticatedRequest<{ workflowId: string }, {}, ManualRunPayload>;
+	type ManualRun = AuthenticatedRequest<
+		{ workflowId: string },
+		{},
+		ManualRunPayload,
+		{ partialExecutionVersion: string | undefined }
```

Comment:
That should be `{ partialExecutionVersion?: string }`, right? 

There is no difference between `?` and `| undefined`. I don't think typescript can type the difference between a property not being set and it being set to `undefined`. 🤔

Reply:
Interesting. I didn't know that TS can differentiate. We should enable it at some point, although it would mean that we'd have to replace this

```ts
const foo = { bar: 1 };

const copy = { ...foo, bar: undefined };
```

with this

```ts
const foo = { bar: 1 };

const copy = { ...foo };
delete copy.bar
```

Anyways, not important for this PR.

Reply:
I think it only matters when _writing_ props. When reading the end result is the same, you receive `undefined` in both cases (the property exists with value undefined and the property doesn't exists)